src/server/server_family.cc

```diff
@@ -1732,6 +1746,42 @@ void PrintPrometheusMetrics(uint64_t uptime, const Metrics& m, DflyCmd* dfly_cmd
                             &resp->body());
   AppendMetricWithoutLabels("evicted_keys_total", "", m.events.evicted_keys, MetricType::COUNTER,
                             &resp->body());
+  // Memory stats
+  string memory_by_class_bytes;
+  if (legacy) {
+    AppendMetricHeader("memory_by_class_bytes", "Memory metrics", MetricType::GAUGE,
+                       &memory_by_class_bytes);
+
+    AppendMetricValue("memory_by_class_bytes", m.lua_stats.used_bytes, {"label"},
+                      {"used_memory_lua"}, &memory_by_class_bytes);
```

Comment:
@romange is this the format that you want ? 

The output as is:

```
dragonfly_memory_by_class_bytes{label="used_memory_lua"} 0
dragonfly_memory_by_class_bytes{label="memory_fiberstack_vms_bytes"} 3817440
dragonfly_memory_by_class_bytes{label="tls_bytes"} 0
dragonfly_memory_by_class_bytes{label="commands_squashing_replies_bytes"} 0
etc...
```

Reply:
we can normalize and simplify labels as well, i.e. `used_memory_lua` -> `used_lua`. 
`tls_bytes` -> `tls`,  `commands_squashing_replies_bytes` -> `squashing_replies`  etc